config.mk

```diff
@@ -1,7 +1,7 @@
 # See LICENSE file for copyright and license details. 
 
 # paths
-PREFIX    = /usr/local
+PREFIX ?= /usr/local
```

Comment:
In my dotfiles, I set `PREFIX=~/.local` to install my non-package-managed programs (without being root) to `~/.local/bin`.